boundary_layer_default_plugin/config/operators/dataproc_cluster_delete.yaml

```diff
@@ -20,15 +20,14 @@ parameters_jsonschema:
     properties:
         cluster_name:
             type: string
+        project_id:
+            type: string
         region:
             type: string
-        priority_weight:
-            type: integer
-            default: 50
-        trigger_rule:
+        gcp_conn_id:
+            type: string
+        delegate_to:
             type: string
-            default: all_done
     required:
         - cluster_name
-        - region
```

Comment:
I removed `region` as required in a few different places because the 1.10.3. operators default it to `global`

Reply:
hey @dossett they actually defaulted to `global` in 1.9 as well, at least on the create and destroy operators.  I would prefer to leave it as a required argument, because this is kind of a dangerous setting (i don't understand why `global` is the default --- do most users really not care which region their clusters are using?).  Dataproc is the only GCP product I'm aware of that doesn't require the region to be specified 🤔